packages/app/src/components/home/HomePage.tsx

```diff
@@ -98,88 +110,77 @@ const HomePage = () => {
 	return (
 		<>
 			<GlobalStyle/>
-			<Grid container spacing={3} justifyContent='space-between'>
-				<Grid item sm={12} md={8}>
-					<Typography variant="h2">BC government developer portal</Typography>
+			<Box sx={{ pb: 1 }}>
+				<Grid container spacing={4} justifyContent='space-between'>
+					<Grid item sm={12} md={8}>
+						<Typography variant="h2">B.C. Government DevHub</Typography>
+					</Grid>
 				</Grid>
-				<Grid item sm={12} md={4}>
-					<HomePageSearchBar
-						classes={{root: classes.searchBar}}
-						InputProps={{classes: {notchedOutline: classes.searchBarOutline}}}
-						placeholder="Search"
-					/> </Grid>
-			</Grid>
-			<Grid container spacing={3} justifyContent='flex-start'>
+			</Box>
+			<Grid container spacing={0} justifyContent='flex-start'>
 				<Grid item xs={12}>
 					<Typography paragraph>
-						Explore our centralized hub to access common resources, tools, documentation and API’s that
-						enable
-						applications and services for the benefit of the developer community. Join us on this journey as
-						we
-						work
-						together to create impactful solutions for people.
-					</Typography>
-					<Typography paragraph>
-						The developer portal is a product of the BC Government Developer Experience team. Help us
-						improve
-						this
-						portal and our other services and tools by <a
-						href="https://github.com/bcgov/developer-portal/issues">providing
-						feedback</a> or <a
-						href="mailto:developer.experience@gov.bc.ca?subject=Developer portal user research">volunteering</a> to
-						participating in user research.
+						The B.C. Government DevHub is a place for developers and product teams to access common resources, tools, technical documentation and APIs.
 					</Typography>
 				</Grid>
 			</Grid>
 
-			<Grid item sm={12}>
+			<Box sx={{ pt: 4, pb: 6 }}>
+				<HomePageSearchBar
+					classes={{root: classes.searchBar}}
+					InputProps={{classes: {notchedOutline: classes.searchBarOutline}}}
+					placeholder="Search"
+				/>
+			</Box>
+
+			<Box sx={{ pb: 1 }}>
 				<Typography variant="h3">
 					Documentation library
 				</Typography>
-			</Grid>
+			</Box>
+
 			<Grid container spacing={3}>
-				<Grid item sm={12} md={4}>
-					<Container>
-						<ContainerImg
-							src="https://digital.gov.bc.ca/wp-content/uploads/2023/03/digitalFrameworkGrey.png"/>
-						<ContainerH2>Get started with mobile development.</ContainerH2>
-						<ContainerP>The BC Mobile Developer Guide provides guidance to developers on steps and practices
-							to
-							follow for developing mobile application for the BC government as well as the supports
-							available to
-							them.</ContainerP>
+				<Grid item sm={12} md={4} style={{display: 'flex'}}>
+					<Container style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-start'}}>
```

Comment:
please use the MUI sx API for this instead next time
https://mui.com/system/flexbox/